src/mlo_dir_conv.cpp

```diff
@@ -130,18 +130,20 @@ static auto GetDirectSolvers()
 
 static auto GetImplicitGemmSolvers()
 {
-    return miopen::solver::SolverContainer<miopen::solver::ConvHipImplicitGemmForwardV4R4Xdlops,
-                                           miopen::solver::ConvHipImplicitGemmV4R4GenXdlopsFwdFp32,
-                                           miopen::solver::ConvHipImplicitGemmV4R4GenFwdXdlops,
-                                           miopen::solver::ConvHipImplicitGemmBwdDataV1R1Xdlops,
-                                           miopen::solver::ConvHipImplicitGemmBwdDataV4R1Xdlops,
-                                           miopen::solver::ConvHipImplicitGemmV4R1Fwd,
-                                           miopen::solver::ConvHipImplicitGemmV4R4Fwd,
-                                           miopen::solver::ConvHipImplicitGemmBwdDataV1R1,
-                                           miopen::solver::ConvHipImplicitGemmBwdDataV4R1,
-                                           miopen::solver::ConvAsmImplicitGemmV4R1DynamicFwd_1x1,
-                                           miopen::solver::ConvAsmImplicitGemmV4R1DynamicFwd,
-                                           miopen::solver::ConvAsmImplicitGemmV4R1DynamicBwd>{};
+    return miopen::solver::SolverContainer<
+        miopen::solver::ConvHipImplicitGemmForwardV4R4Xdlops,
+        miopen::solver::ConvHipImplicitGemmForwardV4R4Xdlops_Padded_Gemm,
```

Comment:
Is this new Solver faster than `ConvHipImplicitGemmForwardV4R4Xdlops`?

Reply:
This solver support some configs that ```ConvHipImplicitGemmForwardV4R4Xdlops``` doesn't support, like the first layer of resnet-50. 

When both solvers are applicable, this solver will be the slower one

Reply:
no, but some configs ConvHipImplicitGemmForwardV4R4Xdlops can't support ,
this kernel can support them, we need both of two

Reply:
So we should put this as the last fwd xdlops solver.